impc_etl/jobs/parse/mgi_gene_pheno_parser.py

```diff
@@ -0,0 +1,58 @@
+"""
+
+[MGI Gene Pheno report](http://www.informatics.jax.org/downloads/reports/MGI_GenePheno.rpt) extraction Task.
+This report contains the list of Gene Phenotype Associations reported in MGI.
+
+### From [MGI Reports / Alleles and Phenotypes section](http://www.informatics.jax.org/downloads/reports/index.html#pheno):
+
+#### 9.  Genotypes and Mammalian Phenotype Annotations for Marker Type Genes
+excluding conditional mutations (tab-delimited):
+
+| Allelic Composition     | Allele Symbol(s) | Allele ID(s) | Genetic Background    | Mammalian Phenotype ID | PubMed ID (pipe-delimited) | MGI Marker Accession ID (pipe-delimited) | MGI Genotype Accession ID (pipe-delimited) |
+|-------------------------|------------------|--------------|-----------------------|------------------------|----------------------------|------------------------------------------|--------------------------------------------|
+| Rb1<tm1Tyj>/Rb1<tm1Tyj> | Rb1<tm1Tyj>      | MGI:1857242  | involves: 129S2/SvPas | MP:0000600             | 12529408                   | MGI:97874                                | MGI:2166359                                |
+...
+"""
+import logging
+
+from airflow.sdk import Variable, asset
+
+from impc_etl.utils.airflow import create_input_asset, create_output_asset
+from impc_etl.utils.spark import with_spark_session
+from impc_etl.utils.utils import extract_tsv
+
+task_logger = logging.getLogger("airflow.task")
+dr_tag = Variable.get("data_release_tag")
+
+mgi_gene_pheno_report_path_asset = create_input_asset("mgi/MGI_GenePheno.rpt")
+mgi_gene_pheno_parquet_asset = create_output_asset("mgi_gene_pheno_parquet")
+
+@asset.multi(
+    schedule=[mgi_gene_pheno_report_path_asset],
+    outlets=[mgi_gene_pheno_parquet_asset],
+    dag_id=f"{dr_tag}_mgi_gene_pheno_parser",
```

Comment:
Add a description to the asset.multi (check [this example](https://github.com/mpi2/impc-etl/blob/d050610619c1a13048e742bb71a2caac59bc6e55/impc_etl/jobs/parse/mgi_mrk_list_parser.py#L35)). You can use the first two lines of the documentation at the top of this file.